packages/client/src/user/userReducer.ts

```diff
@@ -436,11 +436,9 @@ export const userFormReducer: LoopReducer<IUserFormState, UserFormAction> = (
         Cmd.list([
           Cmd.action(clearUserFormData()),
           Cmd.action(
-            goToTeamUserList({
-              id: (action as ISubmitSuccessAction).payload.locationId,
-              searchableText: '',
-              displayLabel: ''
-            })
+            goToTeamUserList(
+              (action as ISubmitSuccessAction).payload.locationId
```

Comment:
I pushed a commit to this branch that fixes all types in `userReducer` so you don't need to cast anymore

Reply:
This level is used in [here](https://www.figma.com/file/HplYu3vGJjAz3u1AmNJ50C/OpenCRVS-Design-Specifications?node-id=4603%3A104481)